extras/gizmo/rotate-gizmo.js

```diff
@@ -174,57 +178,97 @@ class GizmoRotate extends GizmoTransform {
             this._nodeOffsets.clear();
         });
 
-        this.app.on('update', () => {
-            const cameraPos = this.camera.entity.getPosition();
+        app.on('update', () => {
+            const cameraPos = this._camera.entity.getPosition();
             this._faceAxisLookAt(cameraPos);
             this._xyzAxisLookAt(cameraPos);
 
             if (this._dragging) {
-                const gizmoPos = this.gizmo.getPosition();
+                const gizmoPos = this.root.getPosition();
                 this._drawGuideAngleLine(gizmoPos, this._selectedAxis,
                                          this._guideAngleStart, this._guideAngleStartColor);
                 this._drawGuideAngleLine(gizmoPos, this._selectedAxis, this._guideAngleEnd);
             }
         });
     }
 
+    /**
+     * XYZ tube radius.
+     * 
+     * @param {number} value - tube radius.
+     */
```

Comment:
Doesn't quite align with how getters and setters are documented in the core engine src. There, there is just one JSDoc block, whether there's both a get/set or just a get:

https://github.com/playcanvas/engine/blob/main/src/scene/graph-node.js#L350-L374